test/component/Login.test.tsx

```diff
@@ -190,10 +192,12 @@ describe("Login Page Test", () => {
             status: "error",
             message: "Invalid Credentials",
             rucioAccount: "",
+            rucioMultiAccount: "",
             rucioAuthType: "",
             rucioAuthToken: "",
             rucioIdentity: "",
             rucioAuthTokenExpires: "",
+            role: undefined,
         }
 
         await act( async () => render(<LoginStory
```

Comment:
Can we add a test to ensure that the modal shows up when `rucioMultiAccount` is not empty :)

Reply:
done